### ISSUE TITLE
Retrain new data to a pretrained model

### ISSUE THREAD

**Issue by riyu94** [pytorch/pytorch#21263]
Hi I am training data on a lstm model using pytorch
The model works fine when tested.
But i want to improve accuracy if new sequences are detected in data.
I am trying to load this pretrained model and train the model again with new data. 
But it doesn't seem to detect the sequences from the pretrained model (it overwrites the pretrained model)
What steps can I follow to use pretrained model as well as the new trained data for further testing.
Any help appreciated :)

**Comment by soumith** [comment]
please use https://discuss.pytorch.org for questions
